bats/tests/helpers/profile.bash

```diff
@@ -222,13 +222,13 @@ remove_profile_entry() {
 
     case $OS in
     darwin)
-        run profile_plutil -remove "${setting%.}"
+        profile_plutil -remove "${setting%.}" || return
         ;;
     linux)
-        run profile_jq "del(.${setting%.})"
+        profile_jq "del(.${setting%.})" || return
```

Comment:
Since we are now explicit about returning an error, should we make sure that `jq` will throw an error if the setting is not found?

```suggestion
        profile_jq "if has(.${setting%.}) then del(.${setting%.}) else error(\"not found\") end" || return
```